### PR TITLE
Update individual docs + tiny bugfix

### LINKED ISSUE #2527: Update documentation with scout update individual options
I think the command is not covered much in the docs at the moment

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR:
- Adds documentation describing the s`scout update individual` command (fix #2527)
- Fixes a minor bug happening when running the update individual command by specifying right case id but wrong individual name.

**How to test**:
1. Documentation should be easy to read
1. To test the bug fix, use a local instance and run this command both on master and this branch: 
```
scout --demo update individual -c internal_id -n hello
```

**Expected outcome**:
When running the command above from this branch there should be no crash

**Review:**
- [x] code approved by
- [x] tests executed by CR, DN

### CODE REVIEW

**Comment by northwestwitch** [comment]
Test command runned from master branch:

image


From this branch: 🆗 

image

File: docs/admin-guide/updating-individuals.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+| --------------------|:-------------:|
+| bam_file            | path to a bam/cram alignment file                           |
+| mt_bam              | path to a downsampled mitochondrial bam/cram alignment file |
+| rhocall_bed         | path to rhocall output bed file                             |
+| rhocall_wig         | path to rhocall output wig file                             |
+| tiddit_coverage_wig | path to tiddit wig coverage file                            |
+| upd_regions_bed     | path to upd_regions_bed                                     |
+| upd_sites_bed       | path to vcf2cytosure file                                   |
+
+rhocall_bed and rhocall_wig files are both obtained from [rhocall](https://github.com/dnil/rhocall), a software that calls and annotates UPDs from VCF files.
```
Comment (dnil) [inline docs/admin-guide/updating-individuals.md:44]:
```suggestion
rhocall_bed and rhocall_wig files are both obtained from [rhocall](https://github.com/dnil/rhocall), a software that calls and annotates autozygosity from VCF files.
```

File: scout/commands/update/individual.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def individual(case_id, ind, key, value):
         return
     if ind not in individuals:
         click.echo(
-            f"Could not find individual '{ind}' in case individuals. Available individuals for this case: {list(individuals.keys)}"
+            f"Could not find individual '{ind}' in case individuals. Available individuals for this case: {list(individuals.keys())}"
```
Comment (dnil) [inline scout/commands/update/individual.py:43]: 👍

**Review by dnil — APPROVED** [review]

File: scout/commands/update/individual.py (hunk 2 above)
Comment (dnil) [reply scout/commands/update/individual.py:43]: Works here as well!
Screenshot 2021-04-21 at 08 34 32

**Comment by northwestwitch** [comment]
Something's up with the automatic tests this morning. I was thinking there was something wrong with my other MatchMaker refactoring PR, but look at this one as well!

**Comment by northwestwitch** [comment]
Tests are not even starting because of this I think, never saw this before:

image

**Comment by dnil** [comment]
Hm, one in the matrix is ok? 
Screenshot 2021-04-21 at 09 26 36
Something intermittent, or a dependency that updated like just now and doesn't have wheel for a few versions though it claims to somewhere else?

**Comment by northwestwitch** [comment]
No I don't accept this!

image

image

**Comment by dnil** [comment]
> No I don't accept this!

That is the open scientist at heart! "Please show us the data".

**Comment by dnil** [comment]
Did they do something to the pip version or like ubuntu-latest image again? Or why would you suddenly need to import `wheel` for one particular python version?!

**Comment by northwestwitch** [comment]
> Did they do something to the pip version or like ubuntu-latest image again? Or why would you suddenly need to import `wheel` for one particular python version?!

I have no idea

**Comment by northwestwitch** [comment]
It is intermittent, now the list tests are passing!

**Comment by northwestwitch** [comment]
All good now, merging!

**Pull request merged**